EIPS/2020 Block Reward Reduction.md

```diff
@@ -0,0 +1,87 @@
+---
+eip: <to be assigned>
+title: 2020 Block Reward Reduction to 0.5 ETH
+author: John Lilic @jlilic twitter.com/johnlilic johnlilicEIP@gmail.com, Jerome de Tychey @jdetychey twitter.com/jdetychey,  Others
+discussions-to: https://ethereum-magicians.org/t/eip-2878-block-reward-reduction-to-0-5-eth/
+status: Draft
+type: Standards Track
+category Core
+created: 2020-08-11
+---
+
+
+## Simple Summary
+
+This EIP will reduce the block reward paid to proof-of-work validators.  Reducing the block reward will maintain the status quo of periodic community-activated block reward reductions.  
```

Comment:
Please use one space after periods.

Reply:
Is one space a standard for EIP formatting? Or just a personal preference?

Reply:
This is standard EIP formatting.

Reply:
Thanks. We will go through and make sure there is one space after periods.

Reply:
Thank you.